framework/src/meshgenerators/SideSetsGeneratorBase.C

```diff
@@ -104,13 +147,13 @@ SideSetsGeneratorBase::flood(const Elem * elem,
     const std::vector<Point> normals = _fe_face->get_normals();
 
     // We'll just use the normal of the first qp
-    if (std::abs(1.0 - normal * normals[0]) <= _variance)
+    if (normalsWithinTol(normal, normals[0]))
     {
       if (_replace)
         mesh.get_boundary_info().remove_side(elem, side);
 
```

Comment:
I would expect the current  elem and neighbor subdomain element id checks to happen here?

Reply:
same for check boundaries